modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -3,208 +3,125 @@ ifndef::env-site,env-github[]
-
-The structure of the domain depends on whether a wildcard was used in the CN of your certificate, for example:
-
-* A CN with a wildcard (e.g. *.example.com) uses the following request header format: `Host: {app-name}.example.com`. Use the following cURL command to verify:
+... Make sure you have entered values for *Keystore File*, *Keystore Passcode*, *Alias*, and *Key Passcode* before continuing.
+... Select *Deploy*. The *Keystore Passcode* and *Key Passcode* fields are disabled.
+**** If you select a different *Alias* value, the *Key Passcode* field is again enabled.
+**** If you upload a new keystore file, the *Alias* and *Keystore Passcode* fields are again enabled and the *Alias* field contents are cleared.
 +
-----
-curl -Lvk -XGET https://{ip-address}/{path} -H 'Host: {app-name}.example.com'
-----
+The JKS file information is saved in the global secrets group for your organization.
 
-* A CN without a wildcard (e.g. example.com) uses the following request header format: `Host: example.com`. Use the following cURL command to verify:
+** Option 3: *Import from Secrets Manager* (for advanced users)
 +
-----
-curl -Lvk -XGET https://{ip-address}/{app-name}/{path} -H 'Host: example.com'`
-----
-
-== After Enabling Traffic
-
-Runtime Fabric must be configured to route incoming traffic to each enabled application. To complete in order for 
-clients to send requests to deployed applications:
-
-* Configure an external load balancer to load balance HTTPS traffic between each controller VM on Runtime Fabric.
-* Review the advanced options described below when adding an external load balancer.
-* Configure DNS before using the Common Name obtained from the TLS certificate. DNS is required to send 
-requests to applications or API gateways deployed to Runtime Fabric. An "A record" should be added to your 
-DNS provider to map the Common Name to the IP address of the external load balancer or controller VMs.
-
-When deploying new applications or managing existing applications on Runtime Fabric, the Ingress tab is enabled. 
-This tab allows you to specify if inbound traffic should be allowed. After enabling inbound traffic, the default 
-behavior switches to allowed for new application deployments. If there are applications deployed to Runtime 
-Fabric before enabling inbound traffic, they will not receive inbound requests until this setting is enabled.
+This option imports a TLS context from Secrets Manager, and supports advanced configuration such as creating a TLS context, mutual authentication, selecting ciphers, and selecting TLS versions. 
++
+This option is automatically selected for Runtime Fabrics for which inbound traffic was enabled before the PEM and JKS upload options became available. Refer to xref:anypoint-security::configure-tls-context-rtf-lb.adoc[Configure a TLS Context for Runtime Fabric Load Balancer] for instructions.
```

Comment:
```suggestion
This option is automatically selected for instances of Runtime Fabric for which inbound traffic was enabled before the PEM and JKS upload options became available. Refer to xref:anypoint-security::configure-tls-context-rtf-lb.adoc[Configure a TLS Context for Runtime Fabric Load Balancer] for instructions.
```

Reply:
Consider moving this explanation to the beginning of the text for this option: that this used to be the only option but is now for advanced users only?